.github/workflows/test/windows-nightly-ci.yml

```
@@ -0,0 +1,244 @@
+name: (Windows)(nonadmin) Run Playwright tests with latest dev build
+
+on:
+  schedule:
+    - cron: "5 3 * * *"
+  workflow_dispatch:
+
+jobs:
+  start-ec2-instance:
+    uses: ./.github/workflows/provision-runner.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/provision-runner.yml": open /home/jailuser/git/.github/workflows/provision-runner.yml: no such file or directory

(workflow-call)